app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    print(command)
```

Comment:
Clean Code: You have a print statement in your code. If this was used for debugging, please remember to remove it before committing your code.